BusBoard.ConsoleApp/Bus.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BusBoard.ConsoleApp
```

Comment:
The commit message for this commit doesn't really reflect what you've done. It's important to think carefully about when and what you commit and how you describe it.

Reply:
Yeah sorry that's my bad, I did too much without committing so struggled to summarise it